tests/sentry/incidents/test_logic.py

```diff
@@ -1575,6 +1607,37 @@ def test_slack_not_existing(self):
                 integration=integration,
             )
 
+    @responses.activate
+    def test_slack_rate_limiting(self):
```

Comment:
This test asserts the error is handled when updating an existing metric alert rule.

Reply:
You can add these GitHub comments as python comments if you'd like.

Reply:
Will do 👍